contributing/release-schedule.md

```diff
@@ -1,84 +1,81 @@
-- After the cherry pick has been approved, the person currently handling releases (onduty) will work with you to ensure the cherry pick is made.
-- **Once the cherry pick is made you are responsible for verifying that the cherry pick you requested fixes the reported issue and that it does not cause other issues.**
+We are intentionally cautious with our release cadence.
+
+In determining how often we should push new versions of AMP to everyone, we have to weigh many factors including:
+
+- stability for the millions of sites/billions of pages built using AMP
+- cache busting that might happen when we push a new version
+- the desire to get new features out quickly
+
+After considering all of these factors we have arrived at the 1-2 week push cycle. Thus far we have found this to be a reasonable compromise, but we will continue to evaluate all of these factors and may make changes in the future.
 
-**If you are requesting a cherry pick to fix an issue in production** it is very likely you will _also_ need a cherry pick into the canary release since otherwise the problem your cherry pick addresses would reappear as soon as the canary release is pushed to production. Work with the onduty person to determine if you need a cherry pick to both and make sure your cherry pick request issue reflects what you determine.
+### Detailed schedule
 
-## AMP Dev Channel
+We try to stick to this schedule as closely as possible, though complications may cause delays. You can track the latest status about any release in the [_Type: Release_ GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) and the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
 
-The AMP Dev Channel is a way to opt a browser into using the canary release build of the AMP JS libraries. The Dev Channel **may be less stable** and it may contain features not yet available to all users.
+- Tuesday @ [11am Pacific](https://www.google.com/search?q=11am+pacific+in+current+time+zone): new **experimental** and **beta** release builds are created from the [latest master build that passes all of our tests](https://travis-ci.org/ampproject/amphtml/branches) and are pushed to users of AMP who opted into the [AMP Dev Channel](#amp-dev-channel) or [AMP Beta Channel](#amp-beta-channel), respectively.
+- Wednesday: we check bug reports for _Dev Channel_ and _Beta Channel_ users and if everything looks fine, we push the **beta** to 1% of AMP pages
+- Thursday-Monday: we continue to monitor error rates and bug reports for _Dev Channel_ and _Beta Channel_ users and the 1% of pages with the **experimental**/**beta** builds
+- Tuesday the following week: the **beta** build is fully promoted to **stable** (i.e. all AMP pages will now use this build)
 
-Opting into the Dev Channel is great to:
+### AMP Dev Channel (Experimental)
```

Comment:
Because the Dev Channel and Beta Channel are so similar, it probably makes sense to combine these sections into one and just call out that there's an option for enabling experiments or using the stable version.

This can be done in the future PR where you change the name of the "Dev Channel" though.

Reply:
Noted; I have a pending follow-up PR which addresses the name change of these channels around various parts of the docs/codebase. Will make sure you're CC'd